docs/guide/guide_cn.md

```diff
@@ -60,9 +60,11 @@ import (
 	"github.com/cloudwego/netpoll"
 )
 
+var eventLoop netpoll.EventLoop
```

Comment:
I suggest adding the comments about why do we need a global var for eventLoop here. 

I guess the reason likes https://blog.csdn.net/idwtwt/article/details/102985319

Reply:
Agree, but *os.File is indeed saved in the code, see https://github.com/cloudwego/netpoll/blob/main/net_listener.go#L146

I don’t find the exact reason, but debug see eventLoop.svr is nil after runtime.gc

Reply:
I'm very interested at `debug see eventLoop.svr is nil after runtime.gc` , how do you know this variable has not been persisted and be garbage collected.

Reply:
@siddontang https://gist.github.com/joway/faf969ed79f7ffe34f8fe341f16df078 here is a minimum example code you can reproduce this issue. And a fix has been merged: https://github.com/cloudwego/netpoll/pull/54.